### ISSUE TITLE
PR enhancement: automate changed page lists if files are located in `/content/*` 

### ISSUE THREAD

**Issue by juliamrch** [CleverCloud/documentation#346]
### Checking for existing issue

- [X] I have checked for existing [issues](https://github.com/CleverCloud/documentation/issues)

### Describe the content you'd like to see

Use GH actions to create a Hugo centered action to display the url of pages that are modified on a PR, if the charges affects the `/content` folder.

**Comment by Kirbeerus** [comment]
@juliamrch Have you look into how this can be done ?

**Comment by juliamrch** [comment]
@Kirbeerus yes, there are different ways to do it. Will probably make some time in the following days to tackle the `enhancement` tagged issues.
